webhooks/identityconfig.go

```diff
@@ -17,6 +17,7 @@ var (
 type GCPWorkloadIdentityConfig struct {
 	WorkloadIdeneityProvider *string
 	ServiceAccountEmail      *string
+	RunAsUser                int64
```

Comment:
Should we distinguish between "not-set" and "setting to 0"??  It is because `PodSpec.SecurityContext` is [optional](https://github.com/kubernetes/api/blob/9e1815a99d4fa3983f39ae3bf1cd68dda5df49da/core/v1/types.go#L3239). Thus, if securityContext is not set, the default user of container image will be respected.

So, how about making this pointer `int64` --> `*int64`?

Reply:
okey, I didn't use pointer because a pointer to an int64 costs quite similar as the int64, but I get your idea to use is as not set instead of 0. I'll change it today 😄

Reply:
done